The aarch64 dynarec as committed (since c9f233a, 2021-12-22) does not compile: restore the #endif that commit
dropped, flush 'ptr' (this fork has no separate RX mapping for 'ptr_rx'), and store 'reg' where a rename left 'tr'.

diff --git a/mupen64plus-core/src/r4300/new_dynarec/arm64/assem_arm64.c b/mupen64plus-core/src/r4300/new_dynarec/arm64/assem_arm64.c
index 849d886..ad4a289 100644
--- a/mupen64plus-core/src/r4300/new_dynarec/arm64/assem_arm64.c
+++ b/mupen64plus-core/src/r4300/new_dynarec/arm64/assem_arm64.c
@@ -464,7 +464,7 @@ static void *dynamic_linker_ds(void * src, u_int vaddr)
         //assert((ptr2[4]&0xfffffc1f)==0xd61f0000); //br
         add_link(vaddr, ptr2);
         set_jump_target((intptr_t)ptr, (uintptr_t)head->addr);
-        cache_flush((void*)ptr_rx, (void*)(ptr_rx+4));
+        cache_flush((char*)ptr, (char*)ptr+4);
       }
       #ifdef NEW_DYNAREC_DEBUG
       print_debug_info(vaddr);
@@ -3755,7 +3755,7 @@ static void wb_valid(signed char pre[],signed char entry[],u_int dirty_pre,u_int
                 emit_storereg64(reg,HOST_TEMPREG);
               }
               else
-                emit_storereg(tr,hr);
+                emit_storereg(reg,hr);
             } else {
               emit_storereg(reg,hr);
             }
@@ -6729,6 +6729,7 @@ static void do_clear_cache(void)
             }else{
 #ifndef HAVE_LIBNX
               cache_flush((char *)start,(char *)end);
+#endif
               break;
             }
           }
